phoenicis-javafx/src/main/java/com/playonlinux/javafx/views/mainwindow/settings/ViewSettings.java

```diff
@@ -125,15 +125,9 @@ public String fromString(String string) {
     }
 
     public void setSettings(Settings settings) {
-        if (settings.get(Setting.THEME).equals("dark")) {
-            themes.setValue(Themes.DARK);
-        } else if (settings.get(Setting.THEME).equals("breeze")) {
-            themes.setValue(Themes.BREEZE_DARK);
-        } else if (settings.get(Setting.THEME).equals("hidpi")) {
-            themes.setValue(Themes.HIDPI);
-        } else {
-            themes.setValue(Themes.DEFAULT);
-        }
+        final Themes setTheme = Themes.fromShortName(settings.get(Setting.THEME));
```

Comment:
It is really hard to detect what kind of data we have because the view directly queries the model.
We should use a DTO pattern instead